eng/common/superpmi/superpmi.proj

```diff
@@ -0,0 +1,102 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+
+  <PropertyGroup Condition="'$(AGENT_OS)' == 'Windows_NT'">
+    <FileSeparatorChar>\</FileSeparatorChar>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(AGENT_OS)' != 'Windows_NT'">
+    <FileSeparatorChar>/</FileSeparatorChar>
```

Comment:
You don't need this. You can use `MSBuild::NormalizeDirectory` or `MSBuild::NormalizePath` apis. We have a bunch of usages and examples in the runtime repo.

Reply:
Note that this APIs need to be used with full paths, it doesn't work with relative paths.

Reply:
Maybe you do need it... because you're using it to calculate the commands you're going to run on helix.